ERROR at //build/config/compiler/BUILD.gn:2817:9 (//build/toolchain/linux/unbundle:default_for_rust_host_build_tools): Assignment had no effect.
        [ "{{target_out_dir}}/{{label_name}}/{{source_name_part}}.dwo" ]
        ^--------------------------------------------------------------
You set the variable "c_additional_outputs" here and it was unused before it
went out of scope.
See //build/config/BUILDCONFIG.gn:357:3: which caused the file to be included.

This variable was added in https://gn.googlesource.com/gn/+/9641c502e7b394754033dc2ad873a4b3f3a68a6c
in April 2026.


ERROR at //.gn:174:30: Assignment had no effect.
expand_directory_allowlist = build_dotfile_settings.expand_directory_allowlist
                             ^------------------------------------------------
You set the variable "expand_directory_allowlist" here and it was unused before it went out of scope.

This variable was added in https://gn.googlesource.com/gn/+/42ace47bb426ca9705175d866bb9bdf168d5535f
in April 2026.


ERROR at //build/config/BUILDCONFIG.gn:628:30: Assignment had no effect.
        generate_modulemap = default_modulemap_mode
                             ^---------------------
You set the variable "generate_modulemap" here and it was unused before it went
out of scope.

This variable was added in https://gn.googlesource.com/gn/+/6e8dcdebbadf4f8aa75e6a4b6e0bdf89dce1513a
in March 2026.


public_inputs was added in https://gn.googlesource.com/gn/+/7dce53a9983c3f2e4a7f3f06f3d3d30c77493dcd
in July 2026.


--- a/.gn
+++ b/.gn
@@ -175,9 +175,3 @@ exec_script_allowlist =
       # set. If/when a 'proper' GN build for Mesa exists, this can be removed.
       "//third_party/dawn/third_party/meson/meson_target.gni",
     ]
-
-expand_directory_allowlist =
-    build_dotfile_settings.expand_directory_allowlist + [
-      # Requires to enumerate policy template yaml files.
-      "//components/policy/BUILD.gn",
-    ]
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -625,7 +625,6 @@ foreach(_target_type,
       }
 
       if (!defined(generate_modulemap)) {
-        generate_modulemap = default_modulemap_mode
       }
 
       if (use_libcxx_modules) {
@@ -755,7 +754,6 @@ foreach(_target_type,
       }
 
       if (!defined(generate_modulemap)) {
-        generate_modulemap = default_modulemap_mode
       }
 
       if (use_libcxx_modules) {
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -2875,10 +2875,6 @@ config("split_dwarf") {
   # thinlto requires -gsplit-dwarf in ldflags.
   if (use_thin_lto && !is_apple) {
     ldflags = split_dwarf_flags
-  } else {
-    # .dwo files are generated when ThinLTO is not used.
-    c_additional_outputs =
-        [ "{{target_out_dir}}/{{label_name}}/{{source_name_part}}.dwo" ]
   }
 }
 
--- a/third_party/devtools-frontend/src/scripts/build/ninja/copy.gni
+++ b/third_party/devtools-frontend/src/scripts/build/ninja/copy.gni
@@ -25,9 +25,6 @@ template("copy_to_gen") {
       string_join(",", sources),
     ]
 
-    # Expose ts files to dependent targets.
-    public_inputs = filter_include(sources, [ "*.ts" ])
-
     outputs = []
     foreach(_input, sources) {
       outputs += [ "$target_gen_dir/$_input" ]
--- a/third_party/devtools-frontend/src/scripts/build/ninja/devtools_entrypoint.gni
+++ b/third_party/devtools-frontend/src/scripts/build/ninja/devtools_entrypoint.gni
@@ -197,9 +197,6 @@ template("devtools_entrypoint") {
   node_action(target_name) {
     script = "scripts/build/ninja/generate-tsconfig.js"
 
-    # Expose ts files to dependent targets.
-    public_inputs = [ invoker.entrypoint ]
-
     args = [
              rebase_path(_generated_tsconfig_location, root_build_dir),
              rebase_path(invoker.entrypoint, root_build_dir),
--- a/third_party/devtools-frontend/src/scripts/build/ninja/devtools_pre_built.gni
+++ b/third_party/devtools-frontend/src/scripts/build/ninja/devtools_pre_built.gni
@@ -17,9 +17,6 @@ template("devtools_pre_built") {
 
     public_deps = invoker.deps
 
-    # Expose ts files to dependent targets.
-    public_inputs = filter_include(sources, [ "*.ts" ])
-
     _copy_src = rebase_path(get_path_info(target_name, "dir"), root_build_dir)
     _copy_dest = rebase_path(target_gen_dir, root_build_dir)
 
--- a/third_party/devtools-frontend/src/scripts/build/typescript/typescript.gni
+++ b/third_party/devtools-frontend/src/scripts/build/typescript/typescript.gni
@@ -207,6 +207,7 @@ template("ts_library") {
     output_files = [ "$target_gen_dir/$target_name-tsconfig.json" ]
 
     if (defined(sources)) {
+      if (false) {  ### unused
       # Expose ts files to dependent targets.
       public_inputs =
           filter_include(sources,
@@ -217,6 +218,7 @@ template("ts_library") {
                            # and remove them from this list.
                            "*ahem.js",
                          ])
+      }  ### unused
 
       args += [ "--sources" ] + rebase_path(sources, root_build_dir)
 
